pkg/apiserver/debugapi/client.go

```diff
@@ -97,10 +97,11 @@ type pdImplement struct {
 	Client *pd.Client
 }
 
-func (impl *pdImplement) Get(req *Request) ([]byte, error) {
-	return impl.Client.SendGetRequest(req.Path)
+func (impl *pdImplement) Get(req *endpoint.Request) (*httpc.Response, error) {
+	url := fmt.Sprintf("http://%s:%d", req.Host, req.Port)
```

Comment:
This might cause problems under when [cluster TLS](https://docs.pingcap.com/tidb/stable/enable-tls-between-components) is enabled

Reply:
That’s right! I missed that in my todo list.